### ISSUE TITLE
UIAccessibilityPerformEscape doesn't work

### ISSUE THREAD

**Issue by e-sung** [slackhq/PanModal#173]
### Description

When using VoiceOver, UIAccessibilityPerformEscape gesture (two finger scrub) 
doesn't dismiss Panmodal as expected

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [x] bug

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackhq/PanModal/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

---

### Bug Report

Filling out the following details about bugs will help us solve your issue sooner.

#### Reproducible in:

PanModal version: 1.2.7 (latest)

iOS version: iOS 14.7.1, iPhoneXs

#### Steps to reproduce:

1. Turn on VoiceOver
2. Open Panmodal
3. Perform UIAccessibilityPerformEscape gesture

#### Expected result:

Panmodal dismisses

#### Actual result:

Nothing happens

#### Attachments:

https://user-images.githubusercontent.com/4796743/158725232-2883884e-d123-4aad-8864-b580843bdb32.MOV
